### PR TITLE
Install sirius solver with VCPKG, remove antares-deps

### PR DESCRIPTION
**Pull request by sylvlecl**
**Description**

Continues the work of #2078 (which should be merged before in feature branch feature/vcpkg-dependencies), to push further the use of VCPKG, and in particular remove the need for the home-made antares-deps repository.

Hence, this includes:
 - Addition of sirius-solver "overlay" port to install sirius with VCPKG
 - Removal of antares-deps and associated code

### CODE REVIEW

**Comment by sylvlecl** [comment]
Workflow failures are not related to this PR: centos and OL8 buils currently fail on develop (problem with boost version and with nodejs 20).

File: src/packaging/Config.cmake.in [hunk 1]
```diff
@@ -1,8 +1,16 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(ortools)
+find_dependency(ortools REQUIRED)
 find_dependency(minizip-ng)
+if (minizip-ng_FOUND)
```
Comment (flomnes) [inline src/packaging/Config.cmake.in]: This is already done in another PR

**Review by flomnes — APPROVED** [review]
LGTM

**Review by a-zakir — APPROVED** [review]
